docs/reference/flavors/index.md

```diff
@@ -0,0 +1,50 @@
+# Flavors
+
+Any server instance running in {{extra.brand}} Cloud has a **flavor**,
+which defines the number of virtual CPU cores, the amount of virtual
+RAM, and other performance-related factors.
+
+
+## Naming convention
+
+Flavor names in {{extra.brand}} follow a convention, which can be
+summarized as `X.YcZgb`:
```

Comment:
α has been lost here :)

Reply:
No, I deliberately left that out for now because no such public flavors appear to exist, anywhere (yet).

Reply:
Then why to describe tiers that does not have any public flavors? As high-performance local storage will have also α at the end. So then it's kind of inconsistent what we're documenting.

Reply:
I'll defer adding this until a later date.